src/Notebooks/recording_vIsualization.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
Perhaps worth separating the function from the notebook?

Reply:
maybe not, just for convenience changing the colors easily